pinc/Activity.inc

```diff
@@ -95,8 +179,7 @@ class Activity
                 sprintf(_("'%s' pages completed"), $this->id);
         }
 
-        global $Activity_for_id_;
-        $Activity_for_id_[$id] = & $this;
+        Activities::add($this, $id);
```

Comment:
or have 
```php
        Activities::set($this);
```
since the acivity knows its own id.

Reply:
Good point. I originally had this at the top of the function which was before the object knew about its ID.